src/dagmc.cpp

```diff
@@ -847,7 +847,7 @@ void check_dagmc_root_univ()
 
 int32_t next_cell(int32_t surf, int32_t curr_cell, int32_t univ)
 {
-  auto surfp = dynamic_cast<DAGSurface*>(model::surfaces[surf - 1].get());
+  auto surfp = dynamic_cast<DAGSurface*>(model::surfaces[surf].get());
```

Comment:
I see two places that `next_cell` is called:
- particle.cpp: In this case, the value that is passed in is now one less, corresponding with the change here
- plot.cpp: In this case, the value that is passed is still the same, which means the behavior here would be different. Does this matter?

Reply:
Ah yes! Thanks for reminding me of this. I'd call this a fix and it's worth noting in the description here -- I'll update it shortly. I actually wonder if this might be the cause of artifacts in DAGMC models we've been seeing for the ProjectionPlots.

Reply:
It looked like a fix but I thought maybe I was fooling myself :smile: